changelog/21.0/README.md

```diff
@@ -0,0 +1,2 @@
+## v21.0
+* **[21.0.0](21.0.0)**
```

Comment:
We need to regenerate the changelog directory using `go run ./go/tools/releases/releases.go`

Reply:
Fixed